_ml-commons-plugin/connectors.md

```diff
@@ -9,15 +9,54 @@ nav_order: 60
 
 Machine Learning (ML) Connectors provides the ability to integrate OpenSearch ML capabilities with third-party ML tools and platforms. Through connectors, OpenSearch can invoke these third-party endpoints to enrich query results and data pipelines.
 
+You can build connectors in two ways:
+
+1. A [standalone connector](#standalone-connector), saved in a connector index, can be reused and shared with multiple remote models but requires access to both the model and the third party being accessed by the connector, such as OpenAI.
+
+2. An [internal connector](#internal-connector), saved in the model index, can only be used with one remote model. Unlike a standalone connector, users only need access to the model itself to access an internal connector because the connection is established inside the model.
+
+## Integrators
```

Comment:
Should we add this part? Integrator is a persona (some kind of user who use connector framework), not something in connector.

Reply:
Please confirm with Dylan

Reply:
Should we rename this part to blueprints?